### PR TITLE
Add cgamma command

### LINKED ISSUE #94: Suggestion: /cbrightness command
Allows the user to change their brightness/gamma to values higher than is allowed in-game currently, for example; a "fullbright" level, or, something more tame, such as 125%.
**Command(s):** `/cbrightness` `/cgamma`
**Argument(s):** `<integer>`

### PR DESCRIPTION
**Pull request by haykam821**
Fixes #94

### CODE REVIEW

File: src/main/java/net/earthcomputer/clientcommands/command/GammaCommand.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
+package net.earthcomputer.clientcommands.command;
+
+import static com.mojang.brigadier.arguments.DoubleArgumentType.getDouble;
+import static com.mojang.brigadier.arguments.DoubleArgumentType.doubleArg;
+import static net.earthcomputer.clientcommands.command.ClientCommandManager.addClientSideCommand;
+import static net.earthcomputer.clientcommands.command.ClientCommandManager.*;
+import static net.minecraft.server.command.CommandManager.argument;
+import static net.minecraft.server.command.CommandManager.literal;
```
Comment (Earthcomputer) [inline src/main/java/net/earthcomputer/clientcommands/command/GammaCommand.java:8]: These imports need cleaning up. Static imports should always use `*` wildcards, and go after normal imports.

File: src/main/java/net/earthcomputer/clientcommands/command/GammaCommand.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+                .executes(ctx -> setGamma(ctx.getSource(), getDouble(ctx, "gamma")))));
+    }
+
+    private static int setGamma(ServerCommandSource source, double gamma) {
+        MinecraftClient.getInstance().options.gamma = gamma;
+
+        Text feedback = new TranslatableText("commands.cgamma.success", gamma);
+        sendFeedback(feedback);
+
+        return 1;
```
Comment (Earthcomputer) [inline src/main/java/net/earthcomputer/clientcommands/command/GammaCommand.java:33]:
```suggestion
        return 0;
```
Unless you got something more useful to return.

File: src/main/resources/assets/clientcommands/lang/en_us.json [hunk 3]
```diff
@@ -35,6 +35,8 @@
   "commands.cfinditem.starting.keepSearching": "Infinitely searching for %s.",
   "commands.cfinditem.total": "Found %dx %s total",
 
+  "commands.cgamma.success": "Set gamma to %d",
```
Comment (Earthcomputer) [inline src/main/resources/assets/clientcommands/lang/en_us.json:38]:
```suggestion
  "commands.cgamma.success": "Set gamma to %f",
```

**Review by Earthcomputer — CHANGES_REQUESTED** [review]

**Comment by OneYellowLemon** [comment]
@haykam821 Did you see Earth's suggested changes?

File: src/main/java/net/earthcomputer/clientcommands/command/GammaCommand.java (hunk 2 above)
Comment (haykam821) [reply src/main/java/net/earthcomputer/clientcommands/command/GammaCommand.java:33]: Would basing the return value off of whether the gamma was changed work?

Comment (Earthcomputer) [reply src/main/java/net/earthcomputer/clientcommands/command/GammaCommand.java:33]: Sure, although isnt it always changed? I usually return 0 for commands where it makes no sense to have a return value.

Comment (haykam821) [reply src/main/java/net/earthcomputer/clientcommands/command/GammaCommand.java:33]: If the gamma started at 0.3, and the user ran `/cgamma 0.5` twice, the command would return `1` then `0`.

**Comment by Earthcomputer** [comment]
I'll merge this and fix the issues after

**Pull request merged**